### DISCUSSION TITLE
Tracking devices connected in the local network

### DISCUSSION THREAD

**Discussion by fcortijos** [leikoilja/ha-google-home#834] [General]
I use the hagooglewifi HACS, but it is no longer maintained. Can I use ha-google-home for tracking devices connected in the local network?

**Comment by leikoilja** [comment]
> I use the hagooglewifi HACS, but it is no longer maintained. Can I use ha-google-home for tracking devices connected in the local network?

hey, @fcortijos. No, unfortunately not with this integration :)
